src/routers/AppRouter.js

```diff
@@ -71,7 +70,7 @@ const AppRouter = () => {
           path="/project/:projectId/preview"
           component={PreviewPage}
         />
-        <PrivateRoute
+        <Route
```

Comment:
IT will be good to use the OpenRoute component of public routes

Reply:
No need OpenRoute
I removed that component at all